web/models/audit_log.ex

```diff
@@ -0,0 +1,51 @@
+defmodule HexWeb.AuditLog do
+  use HexWeb.Web, :model
+
+  schema "audit_logs" do
+    field :action, :string
+    field :extra_params, :map
```

Comment:
There is always a specific user performing an action so I think the user should be stored in a field rather than in extra_params.